lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -2154,11 +2155,11 @@ TypeSystemSwiftTypeRef::GetBitSize(opaque_compiler_type_t type,
     return {};
   };
   FALLBACK(GetBitSize, (ReconstructType(type), exe_scope));
-  if (exe_scope && exe_scope->CalculateProcess())
+  if (exe_scope && exe_scope->CalculateProcess()) {
```

Comment:
This one is NFC, right? Still a good idea, of course.

Reply:
Probably, I wasn't sure if the whole `VALIDATE_AND_RETURN` would get compiled out or not, if not then this is indeed a NO-OP.